scipy/signal/_filter_design.py

```diff
@@ -366,21 +366,22 @@ def freqz(b, a=1, worN=512, whole=False, plot=None, fs=2*pi,
     --------
     >>> from scipy import signal
     >>> import numpy as np
-    >>> b = signal.firwin(80, 0.5, window=('kaiser', 8))
+    >>> taps, f_c = 80, 1.0  # number of taps and cut-off frequency
+    >>> b = signal.firwin(taps, f_c, window=('kaiser', 8), fs=2*np.pi)
     >>> w, h = signal.freqz(b)
 
     >>> import matplotlib.pyplot as plt
-    >>> fig, ax1 = plt.subplots()
-    >>> ax1.set_title('Digital filter frequency response')
-
-    >>> ax1.plot(w, 20 * np.log10(abs(h)), 'b')
-    >>> ax1.set_ylabel('Amplitude [dB]', color='b')
-    >>> ax1.set_xlabel('Frequency [rad/sample]')
+    >>> fig, ax1 = plt.subplots(tight_layout=True)
+    >>> ax1.set_title(f"Frequency Response of {taps} tap FIR Filter ($f_c={f_c}$ rad/sample)")
```

Comment:
According to the [PEP8 and SciPy] section of the [SciPy Contributor Guide], the line length limit is _88_ characters. Please convert this line into two shorter ones.

[PEP8 and SciPy]: https://docs.scipy.org/doc/scipy/dev/contributor/pep8.html#pep8-scipy
[SciPy Contributor Guide]: https://docs.scipy.org/doc/scipy/dev/contributor/contributor_toc.html